### PR TITLE
[14.0] [NEW] account_no_default: Module to set default_account_id as non-required.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
* Field `default_account_id` will be non-required field in purchase journal.

Closes https://github.com/OCA/purchase-workflow/pull/1055

### CODE REVIEW

**Comment by Chandresh-SerpentCS** [comment]
Ping @dreispt

**Review by dreispt — APPROVED** [review]

File: account_no_default/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "No Default Account",
+    "version": "14.0.1.0.0",
+    "development_status": "Alpha",
```
Comment (HaraldPanten) [inline account_no_default/__manifest__.py]: Just asking... why is it an Alpha? 🤔

**Review by HaraldPanten — APPROVED** [review]
LGTM

File: account_no_default/__manifest__.py (hunk 1 above)
Comment (dreispt) [reply account_no_default/__manifest__.py]: Good point, it should be Beta.

**Pull request merged**